examples/job/wulff-construction-surface-energy-study-cu.py

```diff
@@ -0,0 +1,465 @@
+# In this section, we take the slabs we generated above, and prepare them for the VASP calculation.
+
+# In[]:
+
+
+# Declare the vacuum size we want to have (for easy adjustment by future users)
+vacuum_size = 15
+for miller_index, term_dict in cu_slabs.items():
+    for term, surface in term_dict.items():
+        slab = surface["slab"]
+        # Center the slab's coordinates and adjust the vacuum to vacuum_size
+        # Note that the "vacuum" argument refers to the amount of vacuum on either side - so we divide by 2
+        slab.center(vacuum=vacuum_size/2, axis=2)
+
+        # Write the slab to a file
+        formula = slab.get_chemical_formula(empirical=True)
+        filename = f"{formula}_{miller_index}_term{term}.vasp"
+        ase.io.write(filename, slab)
+
+
+# # Upload POSCARS to Exabyte.io
+# 
+# Now that we have a set of prepared POSCARs, we can upload them to our user account on Exabyte. Let's create a couple material sets to hold these.
+
+# In[]:
+
+
+cu_materials_set = exabyte_materials_endpoint.create_set({"name" : "Webinar_Cu_Wulff_Slabs",
+                                                         "owner": {"_id": ORGANIZATION_ID}})
+
+
+# We want to keep track of material IDs as we go along, so we'll add this key to our Surface dictionary
+
+# In[]:
+
+
+for miller_index, term_dict in cu_slabs.items():
+    for term, surface in term_dict.items():
+        slab = surface["slab"]
+
+        # Get the filename we generated above
+        formula = slab.get_chemical_formula(empirical=True)
+        material_name = f"{formula}_{miller_index}_term{term}"
+        filename = f"{material_name}.vasp"
+
+        # Import the material to Exabyte, and place it in the correct material set
+        with open(filename, "r") as inp:
+            content = inp.read()
+        material_json = exabyte_materials_endpoint.import_from_file(name=material_name,
+                                                                    content=content,
+                                                                    owner_id=ORGANIZATION_ID)
+        exabyte_materials_endpoint.move_to_set(material_json["_id"], "", cu_materials_set["_id"])
+
+        # Adjust our dictionary to keep track of the Material ID
+        surface["material_id"] = material_json["_id"]
+
+
+# # Optimize the Cu Slabs
+# 
+# Now it is time to optimize the Cu slabs. We begin by specifying our INCAR file. It is largely the same as the previous INCAR, except with ISIF set to 2, to avoid changing the cell dimensions.
+
+# In[]:
+
+
+incar_content = ["EDIFF = 1e-6",
+                 "EDIFFG = -0.01",
+                 "ISMEAR = 2",
+                 "IBRION = 2",
+                 "ISIF = 2",
+                 "KPAR = 4",
+                 "NSW = 300",
+                 "ENCUT = 400"]
+kpoints_content = ["Automatic mesh",
+                   "0",
+                   "Gamma",
+                   "8 8 1",
+                   "0 0 0"]
```

Comment:
Tried this without explicitly declaring the k-points file (i.e. letting the platform automatically decide the k-spacing), but got a worse agreement with the literature for these surface energies (both in trends and actual energies). So, I added this back for the current iteration.